core/common/src/main/java/alluxio/underfs/UnderFileSystemRegistry.java

```diff
@@ -111,7 +111,8 @@ public static UnderFileSystem create(String path, Map<String, String> ufsConf) {
     for (UnderFileSystemFactory factory : factories) {
       try {
         // Use the factory to create the actual client for the Under File System
-        return new UnderFileSystemWithLogging(factory.create(path, ufsConf));
+        return new UnderFileSystemWithLogging(
+            factory.create(path, new UnderFileSystemConfiguration(ufsConf)));
```

Comment:
Makes sense to me, however this approach assumes that the user will never need to pass anything other than the `Map`, and any additional fields/logic in the `UfsConf` is able to be populated automatically with server side information.